docs/localnet/docker_localnet.md

```diff
@@ -0,0 +1,46 @@
+# MPC localnet in docker
+
+Brain dump right now, might clean this up for reproducibility.
+
+# Step 1: Build docker images
+
+```shell
+deployment/build-images.sh
```

Comment:
if you don't need the launcher image, then here we could add a parameter to only build the node.
```suggestion
deployment/build-images.sh --node
```

Reply:
Oh nice. That should simplify things.